scripts/ci/run-tests-utils.sh

```diff
@@ -6,6 +6,16 @@ network_setup() {
         done
 }
 
+mem_prof() {
+        echo start memory profiling
+        while(true); do
+                echo trigger memory profiling snapshot
+                sudo pkill -USR2 functionals 2>/dev/null || true
+                sleep 10
+                sudo mv /tmp/skydive-memory.prof /tmp/skydive-memory.prof.$( date +%T ) 2>/dev/null || true
```

Comment:
if you create unique file names using `ioutil.TempFile` you can remove this line

Reply:
no, he would like easily keep and found the memory profiling in time order (+%T)

Reply:
ok